cypress/integration/Dialog.ts

```diff
@@ -0,0 +1,11 @@
+describe('Dialog', () => {
```

Comment:
I agree it would be good to collocate these, but happy for us to look at this in a separate PR.

Reply:
Definitely, I'd prefer to write a few bigger tests first to get a feel for the setup and uncover any existing issues before moving to a monorepo setup.